telegram/bot.py

```diff
@@ -1141,11 +1144,14 @@ def send_contact(self,
             phone_number = contact.phone_number
             first_name = contact.first_name
             last_name = contact.last_name
+            vcard = contact.vcard
```

Comment:
Since vcard is optional I would prefer it if vcard was only added here if it was available in the original contact.